ui/src/app/pages/workspace/workspace-list.spec.tsx

```diff
@@ -116,20 +110,20 @@ describe('WorkspaceList', () => {
       [workspaceOwn.id, WorkspaceAccessLevel.OWNER],
     ]);
 
-    const wrapper = component();
-    await waitOneTickAndUpdate(wrapper);
-    expect(getCardNames(wrapper).length).toEqual(3);
+    component();
+    await waitForNoSpinner();
+    expect(getCardNames().length).toEqual(3);
 
-    await pickAccessLevel(wrapper, 'Reader');
-    expect(getCardNames(wrapper)).toEqual([workspaceRead.name]);
+    await pickAccessLevel('All', 'Reader');
```

Comment:
This doesn't work how I was expecting.  What's happening here?

Reply:
Yeah, that was confusing. It was selecting the dropdown each time by the current value (first parameter)and then changing it to the second parameter.

I updated the dropdown to have an aria-label. Now I find the dropdown once, click it each time we want to change it, and select the value equal to the second parameter.

I think this way is cleaner. Thanks for pointing it out.